### PR TITLE
Resolve #1719

### LINKED ISSUE #1719: Add CLI tests for two new parameters in factory.make_medium
The two new parameters are:
```
--location-ids LOCATION_IDS           REPLACE locations with given ids
                                      Comma separated list of values.
--organization-ids ORGANIZATION_IDS   Comma separated list
                                      of values.

```

### PR DESCRIPTION
**Pull request by abalakh**
Added CLI tests for two new parameters in factory.make_medium

### CODE REVIEW

File: tests/foreman/cli/test_medium.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from robottelo.cli.factory import CLIFactoryError
 from robottelo.test import CLITestCase
 from robottelo.common.decorators import data, run_only_on
-from robottelo.cli.factory import make_medium, make_os
+from robottelo.cli.factory import make_medium, make_os, make_location, make_org
```
Comment (elyezer) [inline tests/foreman/cli/test_medium.py]: A common practice we have is to sort alphabetically the imports, in this case:

```python
from robottelo.cli.factory import make_location, make_medium, make_org, make_os
```

File: tests/foreman/cli/test_medium.py [hunk 2]
```diff
@@ -54,6 +54,54 @@ def test_positive_create_1(self, test_data):
...
 
+    def test_create_medium_with_location(self):
+        """@Test: Check if medium with location can be created
+
+        @Feature: Medium - Positive create
+
+        @Assert: Medium is created and has new location assigned
+
+        """
+        name = gen_string("alpha", 10)
```
Comment (elyezer) [inline tests/foreman/cli/test_medium.py]: `name` variable is used just one time, you can drop it and change line 69 to `'name': gen_string("alpha", 10),`

File: tests/foreman/cli/test_medium.py [hunk 3]
```diff
@@ -54,6 +54,54 @@ def test_positive_create_1(self, test_data):
...
+
+    def test_create_medium_with_organization(self):
+        """@Test: Check if medium with organization can be created
+
+        @Feature: Medium - Positive create
+
+        @Assert: Medium is created and has new organization assigned
+
+        """
+        name = gen_string("alpha", 10)
```
Comment (elyezer) [inline tests/foreman/cli/test_medium.py]: Same here

**Comment by elyezer** [comment]
ACK, pending comments

**Comment by abalakh** [comment]
Updated code according to comments

**Comment by elyezer** [comment]
Looks nice, ACK

**Comment by omaciel** [comment]
ACK

**Pull request merged**